src/runtime_src/doc/toc/p2p.rst

```diff
@@ -151,3 +151,133 @@ PCIe Topology Considerations
 For best performance peer devices wanting to exchange data should be under the same PCIe switch.
 
 If IOMMU is enabled then all peer-to-peer transfers are routed through the root complex which will degrade performance significantly.
+
+
+P2P Data Transfer between FPGA Cards
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+OpenCL coding style
+...................
+
+Consider the example situation as below:
+
+  - P2P data transfer from Card1 to Card2
+  - Source buffer (buf_src) is OpenCL buffer resident of Card1's DDR
+  - Destination buffer (buf_dst) is OpenCL buffer resident of Card2's DDR
+
+Typical coding style:
+
+  1. In the OpenCL host code, create separate `cl_context` for each `cl_device_id`
+  2. Define `buf_src` as regular buffer
+  3. Define `buf_dst` as P2P buffer
+  4. Import the P2P buffer or buf_dst to the context of buf_src. Use the following APIs
+
+       - `xclGetMemObjectFd`
+       - `xclGetMemObjectFromFd`
+  5. Perform the copy operation from `buf_src` to `imported_dst_buf`
+
+.. code-block:: cpp
+
+   // Source Buffer (regular) in source context
+   cl_mem src_buf;
+   src_buf = clCreateBuffer(src_context, CL_MEM_WRITE_ONLY, buffersize, NULL,  &err);
```

Comment:
```suggestion
   src_buf = clCreateBuffer(src_context, CL_MEM_WRITE_ONLY, buffersize, NULL, &err);
```

Reply:
You need to probably add a comment everywhere it is required about not forgetting to test the `err`